src/client/base-client.ts

```diff
@@ -65,7 +65,8 @@ abstract class BaseClient<T extends {}> {
     return this.initialized;
   }
 
-  protected formatResponse<G, O>(
+  // biome-ignore lint/suspicious/noExplicitAny: <explanation>
+  protected formatResponse<G extends Record<string | number, any>, O>(
```

Comment:
```suggestion
  protected formatResponse<G extends Record<string | number, unknown>, O>(
```

what about `unknown` ?

Reply:
I would prefer to use the same type as they use in the library to prevent any strange behaviour

Reply:
ok! in theory it should not give any problems I think 🤔 , but it is fine with me anyway! it was just to avoid suppressing `biome` 👍